pkg/apis/kubermatic/v1/user_project_binding.go

```diff
@@ -62,10 +63,11 @@ type UserProjectBindingSpec struct {
 // +kubebuilder:object:generate=true
 // +kubebuilder:object:root=true
 
-// UserProjectBindingList is a list of users.
+// UserProjectBindingList is a list of KKP users and projects binding.
```

Comment:
`a list of KKP user and project bindings` (bindings is plural, not users/projects).